fix:Google oauth (#983)
appsmithorg/appsmith-docs: website/docs/getting-started/setup/instance-configuration/authentication/google-login.md

## Configure Appsmith instance  

:::caution

> * Please manually test the Client ID and Client secret before disabling the Form login authentication. If the google configuration is invalid, the user is stuck on the login screen. To fix this, you’ll have to update the environment variables manually.

Reviewer: Can we simplify this note? 
```suggestion
* Test the Client ID and Client Secret before disabling Form login on Appsmith. This ensures that users can access Appsmith with their Google credentials. To fix an invalid Google configuration, update the environment variables.
```

> * If you are running appsmith behind an ELB / Proxy, [configure Nginx](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).

Reviewer: These two pieces of information do not tie up together. Maybe a rewrite could be:
```suggestion
* To fix signup issues, if you are running appsmith behind an ELB / Proxy, see [OAuth Sign up not working](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
```
P.S. - can we get this confirmed by the technical team?

* Please manually test the Client ID and Client secret before disabling the Form login authentication. If the google configuration is invalid, the user is stuck on the login screen. To fix this, you’ll have to update the environment variables manually.
* If you are running appsmith behind an ELB / Proxy, [configure Nginx](/help-and-support/troubleshooting-guide/deployment-errors#oauth-sign-up-not-working).
:::

There are two ways to configure the self-hosted Appsmith instance - Admin Settings and Environment Variables:

**On Admin Settings screen (recommended):**  On your self-hosted instance, navigate to the [Admin Settings page](/getting-started/setup/instance-configuration/). 

* Under the **Authentication** tab,  provide the Client ID and Client Secret generated in the previous steps.
* Once you've entered your Client ID and Client Secret, click the **Save & Restart** button at the bottom of the page. This prompts your Appsmith instance with the new configuration in effect.

![](/img/as_google_auth_config.png)

> **Using environment variables:** Update the values for the following keys in your configuration file.

Reviewer: Let's add a short intro for Configuring Appsmith Instance, which tells users that there are below ways in which the instance could be configured:
* Admin Settings(Add links to its subsection)
* Environment Variables(Add links to its subsection)

> On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:

Reviewer: ```suggestion
Navigate to the `docker.env` file located in `<PROJECT_ROOT>/stacks/configuration/`. Update the values for the following keys:
```

> On the machine hosting the Appsmith instance, find your `docker.env` file located in `<project-root>/stacks/configuration/`. Update the values for the following keys:

Reviewer: Create a subhead for Environment Variables.

  # ******************************
   ```

> 

Reviewer: Create a subhead for Admin Settings.
